modules/installation-network-user-infra.adoc

```diff
@@ -159,6 +159,9 @@ the Cluster Version Operator on port `9099`.
 |`10250`-`10259`
 |The default ports that Kubernetes reserves
 
+|`22623`
+| The port handles the machine config server traffic and points to the control plane machines.
```

Comment:
```suggestion
|The port handles traffic from the Machine Config Server and points to the control plane machines.
```
-Delete extra space
- Machine Config Server should be capital
- Small wording suggestion to the first part of sentence. Also, what do you think of changing the last part to "and directs the traffic to the control plane machines"